python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -306,20 +308,22 @@ class RandomForestRegressor(Base):
         cdef size_t params_t64
         if self.n_cols:
             # only if model has been fit previously
-            self.model_pbuf_bytes = self._get_protobuf_bytes()
+            model_pbuf_bytes = self._get_protobuf_bytes()
             params_t = <uintptr_t> self.rf_forest
             rf_forest = \
                 <RandomForestMetaData[float, int]*>params_t
```

Comment:
`rf_forest` and `rf_forest64` should be of type `RandomForestMetaData[float, float]` and `RandomForestMetaData[double double]` respectively